go.mod

```diff
@@ -3,7 +3,7 @@ module github.com/pingcap/tidb
 go 1.20
 
 require (
-	cloud.google.com/go/storage v1.28.1
```

Comment:
Please remember to run ```make bazel_prepare```.

BTW, we have set the pre-commit. so you can install [pre-commit](https://pre-commit.com/) and run ```pre-commit install``` to enable it.

Reply:
If we want to upgrade the latest of this library, you can upgrade the ```.github/dependabot.yml```.

Reply:
> you can upgrade the .github/dependabot.yml

How can I do this? I've checked the file but still know nothing.